code/modules/clothing/neck/_neck.dm

```diff
@@ -193,6 +193,28 @@
 	icon_state = "artist"
 	custom_price = 10
 
+/obj/item/clothing/neck/anti_magic_collar
+	name = "anti-magic collar"
+	desc = "A tight collar used on prisoners to restrict their use of magic, while leaving them vulnerable to it's effects"
+	icon_state = "antimagiccollar"
+	resistance_flags = FIRE_PROOF
+
+/obj/item/clothing/neck/anti_magic_collar/equipped(mob/user, slot, initial = FALSE)
+	. = ..()
+	if(!(slot & slot_flags))
+		ADD_TRAIT(src, TRAIT_NODROP, CURSED_ITEM_TRAIT(type))
```

Comment:
This makes it so if it is equipped to anything but the neck, it will be given Nodrop. Any security officer who buys this item will not be able to take it out of their hands.

Do we want this to even have nodrop in the first place? If so, maybe it should be togglable, maybe through some warden prisoner management console.

Reply:
Updated 
![image](https://github.com/yogstation13/Yogstation/assets/75333826/c5cb8d0c-2b2e-401a-ba66-b85a9fc3fca2)

Reply:
It shouldn't have nodrop until it's locked by the console firstly I think.
You should be able to toggle the lock from the console too, you currently only have it set to unlock once and it becomes useless.

Reply:
I'm not sure about useless, as you can just put it back on someone but I think I see what you're getting at. I'll see what I am able to work out.

Reply:
> It shouldn't have nodrop until it's locked by the console firstly I think. You should be able to toggle the lock from the console too, you currently only have it set to unlock once and it becomes useless.

Don't you think it would be kinda dumb to have the collar not auto-lock on a prisoner you've put it on? It also doesn't really become useless since they still can't cast magic while it's on them, and you can still track it down if they take it ooff

Reply:
Sorry, I forgot it autolocks on every single equip lol. 

I think autolocking is too easy to end up with people trapping themselves unintentionally. It's up to you because I don't think it matters much at the end or he day.

Reply:
I think the unlock via the console is a good alternative for now, since the only people who have access to it are sec/command they *should* have some way to access the console (even via borg!)

Reply:
Well not good alternative but like, good to just leave it as is.

Reply:
nah its fine now